src/index.ts

```diff
@@ -0,0 +1,30 @@
+import * as axs from "./axiosAPI";
+import { InquirerQ } from "./inquirer";
+import { ShipModule, Value } from "./interfaces";
+
+getAPI().then(response => InquirerQ(response));
+
+async function getAPI() {
+  let StarshipUrl: string = `https://swapi.co/api/starships/`;
+  let starShipResponse: ShipModule<Value>[] = [];
+
+  while (StarshipUrl != null) {
+    const starShipsApi = await axs.GetAPIData(StarshipUrl);
+
+    for (let i = 0; i < starShipsApi.data.results.length; i++) {
```

Comment:
`const i` should be enough no need to use `let`

Reply:
cannot set const i as it requires to be incremented, therefore was left as let.